### DISCUSSION TITLE
Performance issue in android device

### DISCUSSION THREAD

**Discussion by gaikoz11** [ppy/osu#20988] [Q&A]
### Type

Performance

### Bug description

It seems in new update the Performance is decrease in android platform. I try all setting but still the same. I try also to edit the resolution in framework.ini of the game in lower resolution but that is not working .( i dont know why ) 😣

My device is : 
Poco x3 pro 
Sd860
8gb ram 
256gb rom 
120hz display

### Screenshots or videos

![IMG_20221010_212054](https://user-images.githubusercontent.com/51250782/194877775-f94e8626-8a04-4f1c-b96b-dbaf9ef49048.jpg)

### Version

2022.1008.0

### Logs

[network.log](https://github.com/ppy/osu/files/9746334/network.log)
[runtime.log](https://github.com/ppy/osu/files/9746337/runtime.log)
[performance.log](https://github.com/ppy/osu/files/9746340/performance.log)
[database.log](https://github.com/ppy/osu/files/9746341/database.log)
[input.log](https://github.com/ppy/osu/files/9746342/input.log)

**Comment by peppy** [comment]
We since pushed a new release with some performance regression fixes. Please update to https://osu.ppy.sh/home/changelog/lazer/2022.1008.2 and see if it helps.

**Reply by gaikoz11** [reply]
The update mutch butter Performance i tested in several days before replying but the ms still have little problem on scrolling the song mune is on 4.2 ms to 18ms / in gameplay stable on 6.3ms to 10ms in 60fps cap ( if im not using 120hz option ) . 

![IMG_20221028_233509](https://user-images.githubusercontent.com/51250782/198676934-47b1ae8a-77eb-4883-a667-69d6d1f6e716.jpg)


This is the ms on scrolling song menu 

![IMG_20221028_233527](https://user-images.githubusercontent.com/51250782/198677035-a0a6da2c-1b33-4524-bdfc-aa9bab93db4c.jpg)


For mobile android or ios not recomended using x8 of infinite refresh rate just use x2 instead

**Comment by SoffiMeower** [comment]
Hey there, adding my own notice to this!
I use a Moto G Stylus 5G, and it still seems quite laggy, having a fluctuating 60fps on Unlimited framerate just on the main menu with the song Triangles. Opening the Song Select also stutters for a bit.

**Comment by peppy** [comment]
Performance and optimisation considerations are an ongoing effort. Especially on android devices, where the hardware can be of varying quality, it may be impossible to run well on some devices. Please be patient and wait for future optimisations, or upgrade your device.

**Reply by V3rald** [reply]
Lazer is out now and my phone is still struggling. in osu!mania i get roughly 45-50 fps and of course phone gets overheated very fast.
My device is Samsung s23 ultra so i am quite shocked to see such performance, was there any change to that since this discussion happened?

Device: Samsung s23 Ultra
SoC/CPU: Qualcomm Snapdragon 8 Gen 2
GPU: Adreno 740
RAM: 8 GB
